pkg/controller/v1beta1/inferenceservice/reconcilers/knative/ksvc_reconciler.go

```diff
@@ -108,6 +109,14 @@ func createKnativeService(componentMeta metav1.ObjectMeta,
 		}
 	}
 
+	// Allow custom annotations for ksvcs that start with serving.knative but not part of serving.knative.dev group name.
+	for aKey, _ := range annotations {
+		if !strings.HasPrefix(aKey, constants.KnativeServingAPIGroupName) && strings.HasPrefix(aKey, constants.KnativeServingAPIGroupNamePrefix) {
```

Comment:
I could but I didnt want to add vendor specific annotations, trying to keep it neutral for future use as well.

Reply:
@skonto I think this is no longer needed as we now allow component level labels and annotations, see https://github.com/kserve/kserve/pull/2925

Reply:
@yuzisun unofrtunately that is per component. The requirement is to set this at the ksvc level, because the OCP Serverless reconciler expects that annotation to be at that level.
For example here is the [diff](https://gist.github.com/skonto/be34e95512337c41a909c49b6ed4007b) between a kserve ksvc and a regular ksvc that works as expected.

Reply:
ok, l'd prefer using a more explicit list, we can add `serving.knative.openshift.io/enablePassthrough` to `managedKsvcAnnotations` ?

Reply:
Yes that is possible. I will update it.

Reply:
@yuzisun done.